README.md

```diff
@@ -1,2 +1,5 @@
-# smoker
-Smoke tests for Foreman and plugins
+# Smoker - Smoke testing the Foreman and friends
+
+Smoker is a [smoke testing](https://en.wikipedia.org/wiki/Smoke_testing_%28software%29) tool aimed at verifying the [Foreman](https://theforeman.org/) functionality and test for regressions. Testing plugins is also within scope, but it should be optional.
```

Comment:
(https://theforeman.org/) 's  <-- 
s/and test for/and testing for/

Reply:
Would it be `the Foreman's functionality` or `Foreman's functionality`?
```suggestion
Smoker is a [smoke testing](https://en.wikipedia.org/wiki/Smoke_testing_%28software%29) tool aimed at verifying the [Foreman](https://theforeman.org/)'s functionality and testing for regressions. Testing plugins is also within scope, but it should be optional.
```

Reply:
as Foreman is the name of the project, a proper noun, you do not need "the". As foreman is a title of a person, its always going to sound a bit odd.

Reply:
I thought so, but glad to have it confirmed :)

Reply:
Updated